.github/workflows/turing.yaml

```diff
@@ -403,6 +405,47 @@ jobs:
             ${{ env.DOCKER_REPOSITORY }}/turing-router:${{ env.TURING_ROUTER_VERSION }}
           docker push ${{ env.DOCKER_REPOSITORY }}/turing-router:${{ env.TURING_ROUTER_VERSION }}
 
+          # Cluster init
+          docker image load --input cluster-init.${{ env.CLUSTER_INIT_VERSION }}.tar
+          docker tag \
+            cluster-init:${{ env.CLUSTER_INIT_VERSION }} \
+            ${{ env.DOCKER_REPOSITORY }}/cluster-init:${{ env.CLUSTER_INIT_VERSION }}
+          docker push ${{ env.DOCKER_REPOSITORY }}/cluster-init:${{ env.CLUSTER_INIT_VERSION }}
+
+      - name: "Install Infrastructure"
+        run: |
+          kubectl create ns infrastructure
+          helm upgrade -n infrastructure turing-init infra/charts/turing-init \
+            --set image.registry=${{ env.LOCAL_REGISTRY }}/ \
+            --set image.repository=${{ github.repository }}/cluster-init \
+            --set image.tag=${{ env.CLUSTER_INIT_VERSION }} \
+            --install \
+            --wait
+
+          # wait for install infra job to finish
+          kubectl logs -n infrastructure -f $(kubectl get po -n infrastructure | grep -v 'NAME' | grep -v 'spark' | head -n 1 | awk '{print $1}')
+          kubectl get po -A
+          kubectl get svc -A
```

Comment:
Is it for debug only ? Can be removed now?

Reply:
I think it's good to have information whenever things go wrong, so that it won't be a black box.